templates/documenso/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.8"
+services:
+  postgres:
+    image: postgres:16
+    networks:
+      - dokploy-network
+    volumes:
+      - documenso-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=documenso
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=documenso
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U documenso"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 10s
+
+  documenso:
+    image: documenso/documenso:latest
```

Comment:
Could you set a version? this in order to prevent to break the applications in the future?